### DISCUSSION TITLE
自己看

### DISCUSSION THREAD

**Discussion by Inc-out** [x-dr/telegraph-Image#85] [General]
![微信图片_20240908000056](https://github.com/user-attachments/assets/57348174-0719-446b-b327-d24532c85d1b)

**Comment by x-dr** [comment]
你有配置tg-channel的环境变量吗？

**Comment by Inc-out** [comment]
我看了半天项目找了半天，没有看见哪里写了配置tg-channel的环境指标，我看了问题提出的，里面的人也说了没有看见哪里写了，希望你把这个在项目最上面更新一下。

x-dr ***@***.***> 于 2024年9月8日周日 09:50写道：

**Comment by Inc-out** [comment]
请问怎么配置tg-channel的环境指标？可以邮件发给我吗？我现在都无法上传

HengmeiliLLC ***@***.***> 于 2024年9月8日周日 09:53写道：

**Reply by x-dr** [reply]
认真看文档
